### ISSUE TITLE
Chip component not working with Google translate

### ISSUE THREAD

**Issue by frec-hansen** [mantinedev/mantine#5586]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.4.0

### What package has an issue?

@mantine/core

### What framework do you use?

Vite

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

`DOMException: Failed to execute 'insertBefore' on 'Node': The node before which the new node is to be inserted is not a child of this node.`

Can easily reproduce this bug by going to: https://mantine.dev/core/chip/, turn on google translate and translate the entire page to another language. Then click on the awesome chip to toggle the select off and on. Then you will be able to see the client side error.

### Possible fix

The issue is probably related to: https://github.com/facebook/react/issues/11538#issuecomment-390386520

TLDR for the fix is to wrap content in a `span` or `div`

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
Fixed in [7.5.1](https://github.com/mantinedev/mantine/releases/tag/7.5.1)
